src/signals/incident-management/containers/IncidentDetail/index.test.js

```diff
@@ -1,10 +1,149 @@
-// import React from 'react';
-// import { shallow } from 'enzyme';
+import React from 'react';
+import { shallow } from 'enzyme';
 
-// import IncidentDetail from 'index';
+import { REQUEST_INCIDENT, PATCH_INCIDENT, REQUEST_ATTACHMENTS, REQUEST_DEFAULT_TEXTS, DISMISS_SPLIT_NOTIFICATION, DISMISS_ERROR } from 'models/incident/constants';
+import { REQUEST_HISTORY_LIST } from 'models/history/constants';
+
+import { IncidentDetail, mapDispatchToProps } from './index';
+
+jest.mock('shared/components/LoadingIndicator', () => () => <div className="loading-indicator" />);
+jest.mock('./components/SplitNotificationBar', () => () => <div className="split-notification-bar" />);
+jest.mock('./components/DetailHeader', () => () => <div className="detail-header" />);
+jest.mock('./components/MetaList', () => () => <div className="meta-list" />);
+jest.mock('./components/History', () => () => <div className="history" />);
+jest.mock('./components/AddNote', () => () => <div className="add-note" />);
+jest.mock('./components/LocationForm', () => () => <div className="location-form" />);
+jest.mock('./components/AttachmentViewer', () => () => <div className="attachment-viewer" />);
+jest.mock('./components/StatusForm', () => () => <div className="status-form" />);
+jest.mock('./components/Detail', () => () => <div className="detail" />);
+jest.mock('./components/LocationPreview', () => () => <div className="location-preview" />);
 
 describe('<IncidentDetail />', () => {
-  it('Expect to have unit tests specified', () => {
-    expect(true).toEqual(true);
+  let wrapper;
+  let props;
+  // eslint-disable-next-line no-unused-vars
+  let instance;
+
+  beforeEach(() => {
+    props = {
+      id: '42',
+      incidentModel: {
+        incident: {
+          id: 42,
+          status: {
+            state: 'm'
+          },
+          category: {
+            category_url: 'foo'
+          }
+        },
+        attachments: [],
+        stadsdeelList: [],
+        changeStatusOptionList: [],
+        statusList: [],
+        defaultTexts: [],
+        priorityList: [],
+        loading: false,
+        patching: {},
+        error: false,
+      },
+      historyModel: {
+        list: []
+      },
+      categories: {},
+      accessToken: '123',
+      baseUrl: 'aaa/',
+
+      onRequestIncident: jest.fn(),
+      onPatchIncident: jest.fn(),
+      onRequestHistoryList: jest.fn(),
+      onRequestAttachments: jest.fn(),
+      onRequestDefaultTexts: jest.fn(),
+      onDismissSplitNotification: jest.fn(),
+      onDismissError: jest.fn()
+    };
+
+    wrapper = shallow(
+      <IncidentDetail {...props} />
+    );
+
+    instance = wrapper.instance();
+  });
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+
+  describe('rendering', () => {
+    it('should render correctly', () => {
+      //  TODO
```

Comment:
Either add an assertion or remove the test case

Reply:
it has a test now